includes/Core/Authentication/Authentication.php

```diff
@@ -1219,7 +1216,7 @@ private function get_proxy_setup_url() {
 		return add_query_arg(
 			array(
 				'action' => Google_Proxy::ACTION_SETUP,
-				'nonce'  => wp_create_nonce( Google_Proxy::ACTION_SETUP ),
+				'nonce'  => $this->google_proxy->get_nonce(),
```

Comment:
See @aaemnnosttv's comment on the issue: The nonce here should be a regular WordPress nonce (i.e. it shouldn't be using the new persistent approach). Only the actual proxy URL (`https://sitekit.withgoogle.com/site-management/setup/...`) should receive that "more persistent" proxy nonce.